### PR TITLE
create card component

### PR DESCRIPTION
**Pull request by git-0r**
# created card component : 
 #  card with badge
card1
 #  card with dismiss
card2
 #  card with text-overlay
card3
 #  card with shadow
card4
#  text only card
card5
#  horizontal card
card6

### CODE REVIEW

File: css/comp_lib_styles/components/card.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  z-index: var(--z-index-three);
+  cursor: pointer;
+  max-width: 60%;
+}
+
+.card-dismiss {
+  right: -1px;
+  top: -1px;
+  padding: 0 0.5rem;
+  border-radius: 0 0.6rem 0 0.6rem;
```
Comment (Sahil25061999) [inline css/comp_lib_styles/components/card.css]: you could use border-radius: 0 0.6rem here

File: css/comp_lib_styles/components/card.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+  background-color: var(--clr-black-disabled);
+}
+
+.card-badge {
+  top: -1px;
+  padding: 0.2rem 0.5rem;
```
Comment (Sahil25061999) [inline css/comp_lib_styles/components/card.css:58]: if value is less than 10px, then no need to use rem you can just use px

File: css/comp_lib_styles/components/card.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  font-weight: var(--fw-bold);
+  color: var(--clr-white-full);
+}
+
+.card-badge-top-right {
+  right: -1px;
+  border-radius: 0 0.6rem 0 0.6rem;
```
Comment (Sahil25061999) [inline css/comp_lib_styles/components/card.css]: This value for border-radius is being repeated so you can just create a variable for it.

**Review by Sahil25061999 — APPROVED** [review]
Apart from mentioned changes, rest looks good,just a UI suggestion you can give different color to button on hover, the design for badge is good, can be merged after the changes.

File: css/comp_lib_styles/components/badge.css [hunk 4]
```diff
@@ -1,99 +1,100 @@
...
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    color: white;
+  border-radius: var(--border-radius-full);
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
```
Comment (aamrapaliwandhre) [inline css/comp_lib_styles/components/badge.css]: use color variable...

File: css/comp_lib_styles/components/card.css [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+}
+
+.card-badge-top-left {
+  left: -1px;
+  border-radius: 0.6rem 0rem 0.6rem 0;
```
Comment (aamrapaliwandhre) [inline css/comp_lib_styles/components/card.css]: you can write 0.6rem 0rem here in border radius..

**Review by aamrapaliwandhre — APPROVED** [review]
other than suggested changes...code looks good to me....

File: css/comp_lib_styles/components/card.css (hunk 2 above)
Comment (git-0r) [reply css/comp_lib_styles/components/card.css:58]: will keep this in mind.

File: css/comp_lib_styles/components/card.css (hunk 1 above)
Comment (git-0r) [reply css/comp_lib_styles/components/card.css]: done!

**Pull request merged**